tests/filecheck/dialects/func/func_invalid.mlir

```diff
@@ -0,0 +1,11 @@
+// RUN: xdsl-opt %s --parsing-diagnostics --verify-diagnostics --split-input-file | filecheck %s
```

Comment:
We could add this directly in `func_ops` using `-split-input-file`

Reply:
Oh, right!

Reply:
Oh, nope AFAIU:
func_ops is roundtripping (`xdsl-opt %s | xdsl-opt --print-op-generic | filecheck %s`)
So this would require heavy Shell shennanigans at the very least (To filecheck the output of the first `xdsl-opt` in case of parser error). Unless I am missing something?

Reply:
Can you not merge the different invalid ones?